accelerator/common_core.c

```diff
@@ -61,11 +101,51 @@ int ta_generate_address(const iota_client_service_t* const service,
 int ta_send_transfer(const iota_client_service_t* const service,
                      const ta_send_transfer_req_t* const req,
                      ta_send_transfer_res_t* res) {
-  /* make bundle
-   * pow
-   * broadcast
-   */
-  return 0;
+  retcode_t ret = RC_OK;
+  flex_trit_t* serialized_txn;
+  find_transactions_req_t* find_tx_req = find_transactions_req_new();
+  find_transactions_res_t* find_tx_res = find_transactions_res_new();
+  hash8019_array_p raw_tx = hash8019_array_new();
+  iota_transaction_t* txn = NULL;
+  bundle_transactions_t* out_bundle = NULL;
+  bundle_transactions_new(&out_bundle);
+  transfer_t* transfers[1];
+  transfers[0] = transfer_data_new(hash243_queue_peek(req->address),
+                                   hash81_queue_peek(req->tag), req->message,
+                                   req->msg_len, (uint64_t)time(NULL));
```

Comment:
[gettimeofday](http://man7.org/linux/man-pages/man2/gettimeofday.2.html) is implemented as a vsyscall on a number of Linux platforms, which is much faster than a normal system call.